ramls/instance.json

```diff
@@ -287,21 +287,6 @@
       },
       "uniqueItems": true
     },
-    "publicationPeriod": {
```

Comment:
please, update Instance class as well to remove manipulations with "publicationPeriod" field there 
https://github.com/folio-org/mod-inventory/blob/master/src/main/java/org/folio/inventory/domain/instances/Instance.java#L77

Reply:
Hi, @RuslanLavrov, Removed, please review, thanks

Reply:
Hi @RomanChernetskyi, thanks!